storage/pruning/pruningStorer.go

```diff
@@ -866,21 +870,22 @@ func (ps *PruningStorer) closeAndDestroyPersisters(epoch uint32) error {
 	}
 	ps.lock.Unlock()
 
-	for _, p := range persistersToClose {
-		err := p.persister.Close()
+	for _, pd := range persistersToClose {
+		//TODO: check if this should return the error or try to close all other persisters first like other methods
```

Comment:
should try to close as much as it can (we can remove the TODO) + keep log.Error id closing fails.

Reply:
changed to log.warn and removed //TODO: